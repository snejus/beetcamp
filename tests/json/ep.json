{
  "@context": "https://schema.org",
  "@id": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 3742084072
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 1
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 4190194561
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 3963823236
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 3742084072
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a3742084072_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "fa010 | Kickdown Vienna"
    },
    {
      "@id": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna#p2928220301",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 2928220301
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "p"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 3963823236
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Vinyl LP"
        },
        {
          "@type": "PropertyValue",
          "name": "image_ids",
          "value": [
            23869838,
            23869839
          ]
        },
        {
          "@type": "PropertyValue",
          "name": "is_music_merch",
          "value": true
        },
        {
          "@type": "PropertyValue",
          "name": "type_id",
          "value": 2
        }
      ],
      "description": "artist: jeånne, DJ Disrespect\nrelease name: Kickdown Vienna\ncatalog: fa010\nbandcamp release date: 09.10.20\nofficial release date: 16.10.20\nformat: vinyl (+ digital only at bandcamp)\n\ntrack info:\nA1 jeånne - the devil's not ås blåck ås he is påinted (hård mix)\nA2 jeånne - the påth to pårådise begins in hell\nB1 DJ DISRESPECT - VIENNA (WARM UP MIX)\nB2 DJ DISRESPECT - TRANSITION (ATHLETIC MIX)\n\nrelease description:\nvienna based artist jeånne maintaining ancient aesthetics on A side. fast pacing acid structures under aggressive beats, enveloped in melancholic strings and melodies.\nDJ DISRESPECT aka Toni Moralez already proven his strong production skills on this and also other labels. He is now delivering two adrenalin kicking, schranz-like dancefloor destroyers out of the depths of Frankfurt am Main.",
      "image": [
        "https://f4.bcbits.com/img/0023869838_10.jpg",
        "https://f4.bcbits.com/img/0023869839_10.jpg"
      ],
      "musicReleaseFormat": "VinylFormat",
      "name": "12\" Vinyl",
      "offers": {
        "@type": "Offer",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "includes_digital_download",
            "value": true
          }
        ],
        "availability": "InStock",
        "price": 9.0,
        "priceCurrency": "EUR",
        "priceSpecification": {
          "minPrice": 9.0
        },
        "url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna#p2928220301-buy"
      }
    },
    {
      "@id": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna#b58140448",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 58140448
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "b"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 3963823236
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 3598959771
        },
        {
          "@type": "PropertyValue",
          "name": "is_bfd",
          "value": true
        }
      ],
      "image": [
        "https://f4.bcbits.com/img/a3598959771_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "full digital discography (20 releases)",
      "offers": {
        "@type": "Offer",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "bundle_size",
            "value": 20
          },
          {
            "@type": "PropertyValue",
            "name": "discount",
            "value": 0.2
          }
        ],
        "availability": "OnlineOnly",
        "price": 163.2,
        "priceCurrency": "EUR",
        "priceSpecification": {
          "minPrice": 163.2
        },
        "url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna#b58140448-buy"
      }
    },
    {
      "@id": "https://fallingapart.bandcamp.com/track/je-nne-the-devils-not-s-bl-ck-s-he-is-p-inted-h-rd-mix",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://fallingapart.bandcamp.com/track/je-nne-the-p-th-to-p-r-dise-begins-in-hell",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://fallingapart.bandcamp.com/track/dj-disrespect-vienna-warm-up-mix",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://fallingapart.bandcamp.com/track/dj-disrespect-transition-athletic-mix",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@type": "MusicGroup",
    "name": "jeånne, DJ DISRESPECT"
  },
  "copyrightNotice": "All Rights Reserved",
  "dateModified": "07 Jul 2023 21:46:09 GMT",
  "datePublished": "09 Oct 2020 00:00:00 GMT",
  "image": "https://f4.bcbits.com/img/a3742084072_10.jpg",
  "keywords": [
    "Chicago",
    "Electronic",
    "Experimental",
    "House",
    "Techno",
    "acid",
    "Frankfurt"
  ],
  "mainEntityOfPage": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
  "name": "fa010 | Kickdown Vienna",
  "numTracks": 4,
  "publisher": {
    "@id": "https://fallingapart.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 3963823236
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "has_download_codes",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 480
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 15313255
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 480
      }
    ],
    "description": "DONT BELIEVE THE HYPE!",
    "foundingLocation": {
      "@type": "Place",
      "name": "Frankfurt, Germany"
    },
    "genre": "https://bandcamp.com/discover/electronic",
    "image": "https://f4.bcbits.com/img/0015313255_10.jpg",
    "mainEntityOfPage": [
      {
        "@type": "WebPage",
        "name": "Instagram",
        "url": "https://www.instagram.com/fallingapartrecords/"
      },
      {
        "@type": "WebPage",
        "name": "discogs.com",
        "url": "https://www.discogs.com/label/1018951-falling-apart"
      },
      {
        "@type": "WebPage",
        "name": "SoundCloud",
        "url": "https://soundcloud.com/fallingapartrecords"
      }
    ],
    "name": "falling apart",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "music",
        "url": "https://fallingapart.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "p"
          }
        ],
        "name": "merch",
        "url": "https://fallingapart.bandcamp.com/merch"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://fallingapart.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://fallingapart.bandcamp.com/track/je-nne-the-devils-not-s-bl-ck-s-he-is-p-inted-h-rd-mix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1568631558
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M25S",
          "mainEntityOfPage": "https://fallingapart.bandcamp.com/track/je-nne-the-devils-not-s-bl-ck-s-he-is-p-inted-h-rd-mix",
          "name": "jeånne - the devil's not ås blåck ås he is påinted (hård mix)"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://fallingapart.bandcamp.com/track/je-nne-the-p-th-to-p-r-dise-begins-in-hell",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1365259276
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M33S",
          "mainEntityOfPage": "https://fallingapart.bandcamp.com/track/je-nne-the-p-th-to-p-r-dise-begins-in-hell",
          "name": "jeånne - the påth to pårådise begins in hell"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://fallingapart.bandcamp.com/track/dj-disrespect-vienna-warm-up-mix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2270629014
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M15S",
          "mainEntityOfPage": "https://fallingapart.bandcamp.com/track/dj-disrespect-vienna-warm-up-mix",
          "name": "DJ DISRESPECT - VIENNA (WARM UP MIX"
        },
        "position": 3
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://fallingapart.bandcamp.com/track/dj-disrespect-transition-athletic-mix",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1411519787
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M33S",
          "mainEntityOfPage": "https://fallingapart.bandcamp.com/track/dj-disrespect-transition-athletic-mix",
          "name": "DJ DISRESPECT - TRANSITION (ATHLETIC MIX)"
        },
        "position": 4
      }
    ],
    "numberOfItems": 4
  }
}
