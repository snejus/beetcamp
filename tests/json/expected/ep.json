[
  {
    "album": "Kickdown Vienna",
    "album_id": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "album",
    "albumtypes": "album",
    "artist": "DJ DISRESPECT, jeånne",
    "artist_credit": null,
    "artist_id": "https://fallingapart.bandcamp.com",
    "artist_sort": null,
    "asin": null,
    "catalognum": "fa010",
    "comments": null,
    "country": "DE",
    "data_source": "bandcamp",
    "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
    "day": 9,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": "experimental, house, techno",
    "label": "falling apart",
    "language": null,
    "media": "Digital Media",
    "mediums": 1,
    "month": 10,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": "electronic",
    "tracks": [
      {
        "arranger": null,
        "artist": "jeånne",
        "artist_credit": null,
        "artist_id": "https://fallingapart.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
        "disctitle": null,
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 385,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 4,
        "release_track_id": null,
        "title": "the devil's not ås blåck ås he is påinted (hård mix)",
        "track_alt": null,
        "track_id": "https://fallingapart.bandcamp.com/track/je-nne-the-devils-not-s-bl-ck-s-he-is-p-inted-h-rd-mix",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "jeånne",
        "artist_credit": null,
        "artist_id": "https://fallingapart.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
        "disctitle": null,
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 333,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 4,
        "release_track_id": null,
        "title": "the påth to pårådise begins in hell",
        "track_alt": null,
        "track_id": "https://fallingapart.bandcamp.com/track/je-nne-the-p-th-to-p-r-dise-begins-in-hell",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "DJ DISRESPECT",
        "artist_credit": null,
        "artist_id": "https://fallingapart.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
        "disctitle": null,
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 315,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 4,
        "release_track_id": null,
        "title": "VIENNA (WARM UP MIX)",
        "track_alt": null,
        "track_id": "https://fallingapart.bandcamp.com/track/dj-disrespect-vienna-warm-up-mix",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "DJ DISRESPECT",
        "artist_credit": null,
        "artist_id": "https://fallingapart.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
        "disctitle": null,
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 333,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 4,
        "release_track_id": null,
        "title": "TRANSITION (ATHLETIC MIX)",
        "track_alt": null,
        "track_id": "https://fallingapart.bandcamp.com/track/dj-disrespect-transition-athletic-mix",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2020
  },
  {
    "album": "Kickdown Vienna",
    "album_id": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna#p2928220301",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "album",
    "albumtypes": "album",
    "artist": "DJ DISRESPECT, jeånne",
    "artist_credit": null,
    "artist_id": "https://fallingapart.bandcamp.com",
    "artist_sort": null,
    "asin": null,
    "catalognum": "fa010",
    "comments": "artist: jeånne, DJ Disrespect\nrelease name: Kickdown Vienna\ncatalog: fa010\nbandcamp release date: 09.10.20\nofficial release date: 16.10.20\nformat: vinyl (+ digital only at bandcamp)\n\ntrack info:\nA1 jeånne - the devil's not ås blåck ås he is påinted (hård mix)\nA2 jeånne - the påth to pårådise begins in hell\nB1 DJ DISRESPECT - VIENNA (WARM UP MIX)\nB2 DJ DISRESPECT - TRANSITION (ATHLETIC MIX)\n\nrelease description:\nvienna based artist jeånne maintaining ancient aesthetics on A side. fast pacing acid structures under aggressive beats, enveloped in melancholic strings and melodies.\nDJ DISRESPECT aka Toni Moralez already proven his strong production skills on this and also other labels. He is now delivering two adrenalin kicking, schranz-like dancefloor destroyers out of the depths of Frankfurt am Main.",
    "country": "DE",
    "data_source": "bandcamp",
    "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
    "day": 9,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": "experimental, house, techno",
    "label": "falling apart",
    "language": null,
    "media": "Vinyl",
    "mediums": 1,
    "month": 10,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": "electronic",
    "tracks": [
      {
        "arranger": null,
        "artist": "jeånne",
        "artist_credit": null,
        "artist_id": "https://fallingapart.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
        "disctitle": "12\" Vinyl",
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 385,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 4,
        "release_track_id": null,
        "title": "the devil's not ås blåck ås he is påinted (hård mix)",
        "track_alt": "A1",
        "track_id": "https://fallingapart.bandcamp.com/track/je-nne-the-devils-not-s-bl-ck-s-he-is-p-inted-h-rd-mix",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "jeånne",
        "artist_credit": null,
        "artist_id": "https://fallingapart.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
        "disctitle": "12\" Vinyl",
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 333,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 4,
        "release_track_id": null,
        "title": "the påth to pårådise begins in hell",
        "track_alt": "A2",
        "track_id": "https://fallingapart.bandcamp.com/track/je-nne-the-p-th-to-p-r-dise-begins-in-hell",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "DJ DISRESPECT",
        "artist_credit": null,
        "artist_id": "https://fallingapart.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
        "disctitle": "12\" Vinyl",
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 315,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 4,
        "release_track_id": null,
        "title": "VIENNA (WARM UP MIX)",
        "track_alt": "B1",
        "track_id": "https://fallingapart.bandcamp.com/track/dj-disrespect-vienna-warm-up-mix",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "DJ DISRESPECT",
        "artist_credit": null,
        "artist_id": "https://fallingapart.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://fallingapart.bandcamp.com/album/fa010-kickdown-vienna",
        "disctitle": "12\" Vinyl",
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 333,
        "lyricist": null,
        "mb_workid": null,
        "media": "Vinyl",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 4,
        "release_track_id": null,
        "title": "TRANSITION (ATHLETIC MIX)",
        "track_alt": "B2",
        "track_id": "https://fallingapart.bandcamp.com/track/dj-disrespect-transition-athletic-mix",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2020
  }
]
